src/main/java/edu/kis/powp/jobs2d/drivers/adapter/CustomizableLineDrawerAdapter.java

```diff
@@ -0,0 +1,41 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+import edu.kis.powp.jobs2d.lines.CustomLine;
+
+import java.awt.*;
+
+public class CustomizableLineDrawerAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
The name is better but it lacks a driver aspect.

Inheritance is not used (no super drawLine calls). Should be removed!